5-infrastructure/modules/base-env/boa-gke.tf

```diff
@@ -116,7 +117,7 @@ module "clusters" {
     "mesh_id" = "proj-${data.google_project.gke_project.number}"
   }
   node_pools_tags = {
-    "np-${each.value.region}" : ["boa-${each.key}-cluster", "allow-google-apis"]
+    "np-${each.value.region}" : ["boa-${each.key}-cluster", "allow-google-apis", "egress-internet"]
```

Comment:
lets add a common `boa-cluster` tag across all clusters, this way we can target via single tag in `target_tags`

Reply:
the seperate tags allow me to target one specific cluster in fw rules, but I can add a 4th tag "boa-cluster"

Reply:
sorry I meant in addition to existing as well, that sgtm